Doppler.Sap/Factory/CreditNoteHandler.cs

```diff
@@ -95,6 +185,74 @@ private async Task<SapTaskResult> CreateCreditNote(SapSaleOrderInvoiceResponse s
             };
         }
 
+        private async Task<SapTaskResult> UpdateCreditNote(SapCreditNoteResponse sapCreditNoteResponse, SapTask dequeuedTask, string sapSystem)
+        {
+            var sapCreditNote = GetMapper(sapSystem).MapToSapCreditNote(dequeuedTask.CreditNoteRequest);
+            var serviceSetting = SapServiceSettings.GetSettings(_sapConfig, sapSystem);
+            var uriString = $"{serviceSetting.BaseServerUrl}{serviceSetting.BillingConfig.CreditNotesEndpoint}({sapCreditNoteResponse.DocEntry})";
+            var sapResponse = await SendMessage(sapCreditNote, sapSystem, uriString, HttpMethod.Patch);
+
+            var taskResult = new SapTaskResult
+            {
+                IsSuccessful = sapResponse.IsSuccessStatusCode,
+                SapResponseContent = await sapResponse.Content.ReadAsStringAsync(),
+                TaskName = "Updating Credit Note"
+            };
+
+            return taskResult;
+        }
+
+        private async Task<SapTaskResult> CancelCreditNote(SapCreditNoteResponse sapCreditNoteResponse, SapTask dequeuedTask, string sapSystem)
+        {
+            var sapCreditNote = GetMapper(sapSystem).MapToSapCreditNote(dequeuedTask.CancelCreditNoteRequest);
```

Comment:
Why this variable is not used?

Reply:
I will remove the variable is unnecessary

Reply:
Done